contracts/SpokeRegistry.sol

```diff
@@ -2,12 +2,15 @@
 pragma solidity ^0.6.12;
 
 contract SpokeRegistry {
+    event SpokeRegistered(uint256 spokeID, address spokeContract);
+
     mapping(uint256 => address) public registeredSpokes;
     uint256 public numSpokes;
 
     function registerSpoke(address spokeContract) external {
         numSpokes++;
         registeredSpokes[numSpokes] = spokeContract;
+        emit SpokeRegistered(numSpokes, spokeContract);
     }
```

Comment:
Would be nice to have a fast test case for this function and the new event, but given that there is not an existing case will leave at author's discretion.